app/scripts/directives/resources.js

```diff
@@ -1,6 +1,65 @@
 'use strict';
 
 angular.module('openshiftConsole')
+  .directive('containerStatuses', function($filter) {
+    return {
+      restrict: 'E',
+      scope: {
+        pod: '=',
+        startDebugTerminal: '=?',
+        detailed: '=?'
+      },
+      templateUrl: 'views/_container-statuses.html',
+      link: function(scope) {
+        scope.hasDebugTerminal = angular.isFunction(scope.startDebugTerminal);
+
+        var isContainerTerminated = $filter('isContainerTerminatedSuccessfully');
```

Comment:
I'd keep `Successfully` in the name since this should return false if it terminated with a failure.